.travis.yml

```diff
@@ -1,6 +1,6 @@
 # although Lmod is written in Lua, use Python environment
 language: python
-sudo: false
+sudo: required
```

Comment:
Not that I know of. R is a community supported language. The r-travis [script](https://github.com/eddelbuettel/r-travis/blob/master/run.sh) adds apt-repositories to provide `r-base` etc... and uses `sudo` multiple times.